distribution/docs/src/main/resources/_contents/_installing/installing-intro-contents.adoc

```diff
@@ -0,0 +1,357 @@
+
+[IMPORTANT]
+====
+Although ${branding} can be installed by any user, it is recommended for security reasons to have a non-`root` user execute the ${branding} installation.
+====
+
+=== ${branding} Installation Prerequisites
+
+These are the system/environment requirements to configure _prior_ to an installation.
+
+==== Directory Permissions
```

Comment:
It is confusing. I guess it depends on whether it's really a prerequisite to installation. Does it work to change permissions on the HOME directory, into which these others will be created? Is it something you'd realistically do _before_ installing anything? It doesn't read like a prior preparation step to me.